be/src/column/fixed_length_column_base.h

```diff
@@ -183,6 +183,8 @@ class FixedLengthColumnBase : public ColumnFactory<Column, FixedLengthColumnBase
         _data.clear();
     }
 
+    bool reach_capacity_limit() const override { return _data.size() >= UINT32_MAX; }
```

Comment:
Would it be better to add a new constant named max_capacity_limit?

Reply:
Fixed